mkdocs/docs/how-to-release.md

```diff
@@ -205,36 +258,46 @@ The release candidate has been accepted as PyIceberg <VERSION>. Thanks everyone,
 Kind regards,
 ```
 
-### Copy the artifacts to the release dist
+### Upload the accepted RC to Apache Release SVN
+<!-- prettier-ignore-start -->
 
-```bash
-export RC=rc2
-export VERSION=0.7.0${RC}
-export VERSION_WITHOUT_RC=${VERSION/rc?/}
+!!! note
+    Only a PMC member has the permission to upload an artifact to the SVN release dist.
+
+<!-- prettier-ignore-end -->
 
+```bash
 export SVN_DEV_DIR_VERSIONED="https://dist.apache.org/repos/dist/dev/iceberg/pyiceberg-${VERSION}"
 export SVN_RELEASE_DIR_VERSIONED="https://dist.apache.org/repos/dist/release/iceberg/pyiceberg-${VERSION_WITHOUT_RC}"
 
 svn mv ${SVN_DEV_DIR_VERSIONED} ${SVN_RELEASE_DIR_VERSIONED} -m "PyIceberg: Add release ${VERSION_WITHOUT_RC}"
 ```
 
+Verify the artifact is uploaded to [https://dist.apache.org/repos/dist/release/iceberg](https://dist.apache.org/repos/dist/release/iceberg/).
```

Comment:
Can we add a step here to only host the lastest release on the `dist/release/`. I recently cleaned it up, but it would be good to make this part of the release itself :)